app/components/utility/filter_component.html.erb

```diff
@@ -38,6 +38,7 @@
         <% end %>
         <div class="moj-filter__options">
           <form method="get">
+           <%= hidden_field_tag(:resolved, params[:resolved]) if params[:resolved].present? %>
```

Comment:
Do you think is worth adding system specs for search in each tab?

Reply:
Discussed offline - This has been covered